src/08-tasks/tasks.apib

```diff
@@ -187,3 +187,21 @@ Delete a task.
         + id: `5f0afd8a-8a40-48a4-bbe6-7d0e9c61bb6d` (string, required)
 
 + Response 204
+
+### tasks.schedule [POST /tasks.schedule]
+
+Schedule a task.
```

Comment:
```suggestion
Schedule a task in your calendar.
```

Reply:
We have no endpoint for calendars (yet), but we have a reference to calendar on the API Docs, so i believe this have some value, giving more clean context.